src/docs/concepts-development/content-management/index.md

```diff
@@ -0,0 +1,245 @@
+---
+title: Content management in Flex
+slug: content-management
+updated: 2023-01-11
+category: concepts-development
+ingress:
+  This article introduces the Flex content management system and how
+  static pages can be managed in Flex Console
```

Comment:
Comment:

Come to think of it, I'm not sure if _static_ is the term we want to use to describe pages dynamically managed through Pages. "A CMS allows you to build static content pages" is a bit of an oxymoron.
 Should we instead use "content page" instead of "static page"?
 
 @SariSaar